### PR TITLE
{bio}[intel/2017b] pybedtools v0.7.10, Pysam v0.13, BEDTools v2.26.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/08c5a75091ea804fafb395d76cd216ac for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/10496182b92bedef3bbc9364e00798d7 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/455d01e6e481dcb031ad261006b2879e for a full test report.

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.26.0-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.26.0-intel-2017b.eb]: @boegel - please update my affiliation here and in the next one.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**